detection.py

```diff
@@ -95,6 +95,8 @@ def _create_frequency_arrays(
             {key: Counter(new)[key] for key in base_cat_freq.keys()}
         )
 
+        self._check_missing_values_proportions(baseline, new)
```

Comment:
The method shouldn't be called here since the missing values test is also relevant for numerical values. I suggest calling the method in `detect_single_drift` and change there and in `detect_drift` the code so that a message is displayed in case of a distribution change in missing values (in addition to the regular distribution drift)